.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: csharp
+mono: none
+dotnet: 2.0
```

Comment:
I think this needs to be `2.0.0`

right now travis is parsing this as a number but expecting a string, which I think may be why the build is erroring.